src/test/java/com/lsd/report/approval/LsdContextTest.java

```diff
@@ -69,7 +69,7 @@ void createsReportWithScenariosAndEvents() {
         lsdContext.addFact("some important value", "123456");
         lsdContext.completeScenario("An Error scenario", "<p>Failure! Expected value to be 123 but was 123456</p>", ERROR);
 
-        Approvals.verify(lsdContext.completeReport("Approval Report").toFile());
+        Approvals.verifyHtml(lsdContext.generateReport("Approval Report"));
```

Comment:
@nickmcdowall Are you OK with this change? And same below...

Reply:
Hey, I recall the generateReport was added ages ago but was it just for testing purposes (can't remember). I guess it should be fine but maybe the completeReport method needs changing so we don't need to expose other public methods to get the report html.. but let's go ahead for now until that can be figured out I guess.

Reply:
I think it was added for the lsd-ui project, not only for testing.